bokeh/models/graphs.py

```diff
@@ -174,6 +175,19 @@ class NodesAndAdjacentNodes(GraphHitTestPolicy):
 
     pass
 
+class NodesAndLinkedEdgesAndLinkedNodes(GraphHitTestPolicy):
+    '''
+    With the ``NodesAndLinkedEdgesAndLinkedNodes`` policy, inspection or selection
+    of graph nodes will result in the inspection or selection of the node, any their
+    linked graph edges, and of the linked graph nodes. There is no selection or
+    inspection of graph edges, and no indication of which node is the tool-selected
+    one from the policy-selected nodes. There is no selection or inspection of graph
+    edges, and no indication of which node is the tool-selected one from the
+    policy-selected nodes.
```

Comment:
```suggestion
    With the ``NodesAndLinkedEdgesAndLinkedNodes`` policy, inspection or selection
    of graph nodes will result in the inspection or selection of the node, any
    linked graph edges, and any adjacent graph nodes. There is no selection or
    inspection of graph edges, and no indication of which node is tool-selected.
```
Suggestion to bring down the word count and use "adjacent" to remain consistent with my other comment of renaming to `NodesAndLinkedEdgesAndAdjacentNodes`